src/main/java/wolox/training/exceptions/BookNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package wolox.training.exceptions;
+
+public class BookNotFoundException extends RuntimeException {
+
+    public BookNotFoundException() {
+        super();
+    }
+    public BookNotFoundException(String message, Throwable cause) {
```

Comment:
I think maybe it's not necessary to provide this constructor(`public BookNotFoundException(String message, Throwable cause)`) because there is only one reason to throw this exception and it's because the book was not found. So maybe you can have the first constructor and do something like this:

```
    public BookNotFoundException() {
        super("The book doesn't exist");
    }
```